src/digitalmarketplace/components/alert/alert.yaml

```diff
@@ -0,0 +1,54 @@
+params:
+- name: type
+  type: string
+  required: true
+  description: Defines which type of alert will be rendered.
```

Comment:
Could benefit from giving the options here Ie:

> one of (success, error, notice)

Reply:
Doing in the next commit.